lib/common_test/test_server/ts.erl

```diff
@@ -182,10 +182,6 @@ help(installed) ->
 	 "  verbose           - Same as {verbose, 1}\n",
 	 "  {vars, Vars}      - Variables in addition to the 'variables' file\n",
 	 "                      Can be any of the install options\n",
-	 "  {trace, TraceSpec}- Start call trace on target and slave nodes\n",
```

Comment:
I might have misinterpreted this option. I could not find documentation on this feature, and assumed it is related to "tracer node". To be honest, if this feature is hard to discover and is not actively used, I'd rather remove it too. But if you prefer to keep it, I can adjust the PR and keep line 185 and lines 198-205.

I also think that most parts of `test_server_node.erl` will be also gone after https://github.com/erlang/otp/pull/5162 is merged and I refactor remaining tests to use `?CT_PEER` instead of `test_server:start_node`.

Reply:
I'm guessing it _could be_ related to tracer node in the past. AFAIK, modules ts and prefixed with "ts_" are used internally in OTP test runs.

If you agree ts doc change is unrelated to #5021, please leave it as it is. I can fix the doc myself, once I play more with discovered feature.

Rest of PR looks OK.

Regarding #5162 - I haven't looked into it yet.